SPECS/liburing/liburing.spec

```diff
@@ -1,16 +1,12 @@
 Summary:        Linux-native io_uring I/O access library
 Name:           liburing
-Version:        2.0
```

Comment:
Are there incompatible API or ABI changes with this package update?  Are we confident that the packages that depend on this will still run if we take this change?

Reply:
Debian's liburing2 corresponding to soname 2 has been active since the 2.0 release, with soname 1 (liburing1) terminating at 0.7. so at least by Debian standards, according to the liburing maintainer (Guillem Jover), there have been no ABI changes since 2.0.

https://github.com/axboe/liburing/issues/724 looks somewhat dubious, but given that Jens Axboe has given no indication that he'll be bumping the SOVERSION, apparently he feels the removed symbols to not have been covered by ABI guarantees.

I know we're shipping 2.3 in Debian without major problems.

Reply:
i just checked and neither fedora nor debian have seen any FTBFS due to this update. they have a larger package base than mariner, though i'm sure there are some packages unique to mariner. it appears that ABI changes, if you can call them that, were limited to function removals, which would lead to pretty obvious failures in an archive-wide build (which i believe was done).